roles/ceph-common/tasks/installs/debian_community_repository.yml

```diff
@@ -10,14 +10,12 @@
     state: present
   changed_when: false
 
-- name: configure debian gluster nfs ganesha community repository for rgw nfs
+- name: add nfs-ganesha stable repository
   apt_repository:
-    repo: "{{ item }}"
+    repo: "deb {{ nfs_ganesha_stable_deb_repo }} {{ ceph_stable_distro_source | default(ansible_lsb.codename) }} main"
     state: present
-  with_items:
-    - ppa:gluster/libntirpc
-    - ppa:gluster/nfs-ganesha
   changed_when: false
-  when:
-    - (nfs_obj_gw or nfs_file_gw)
-    - not ansible_distribution == "Debian"
+  when: 
+    - nfs_ganesha_stable is defined
```

Comment:
no need for this check as the var must be defined in ceph-common/defaults/main.yml